beacon_node/beacon_chain/src/data_availability_checker.rs

```diff
@@ -445,8 +445,6 @@ impl<T: BeaconChainTypes> DataAvailabilityChecker<T> {
             .flatten()
             .map(CustodyDataColumn::into_inner)
             .collect::<Vec<_>>();
-        let all_data_columns =
-            RuntimeVariableList::from_vec(all_data_columns, T::EthSpec::number_of_columns());
```

Comment:
What was this line _trying_ to do? I guess it was assuming these columns were for a single block and coercing them to a `RuntimeVariableList` in order to "check" the length?

Reply:
Ohh just saw you made an issue 🙏

Reply:
PR here:

- https://github.com/sigp/lighthouse/pull/7930

Reply:
> What was this line _trying_ to do? I guess it was assuming these columns were for a single block and coercing them to a `RuntimeVariableList` in order to "check" the length?

Yeah i think exactly this i think